lldb/source/Plugins/Language/ObjC/NSSet.cpp

```diff
@@ -395,8 +395,8 @@ lldb_private::formatters::NSSetISyntheticFrontEnd::GetIndexOfChildWithName(
       (idx < UINT32_MAX && idx >= CalculateNumChildrenIgnoringErrors()))
     return llvm::createStringError(
         "'SyntheticChildrenFrontEnd::NSSetISyntheticFrontEnd' cannot find "
-        "index of child '%s'",
-        name.AsCString());
+        "index of child '%s'. (idx='%d')",
```

Comment:
```suggestion
        "index of child '%s'. (idx='%" PRu32 "')",
```
(same goes for the other `idx` logs)